### DISCUSSION TITLE
What is populating style-editor.css?

### DISCUSSION THREAD

**Discussion by mikedefieslife** [gregsullivan/_tw#53] [Q&A]
**Solved**

I'm find that the base theme applies 40rem max-width to entry content, and sets margins to auto. 

This shows up in style-editor.css file 

That file seems to be put together from data elsewhere as it regenerates each time the TW and other config files are altered. 

Despite the commenting I can't for life of me find where these post and layout styles are coming from. Especially as when I check /custom/components/ all the .css files in there are basically empty. 

So what is populating style-editor.css? It's not the theme.json, as changing values there doesn't have any effect. Nor does using .alignwide or .alignful

**EDIT: Found it in comments.css**

**Comment by gregsullivan** [comment]
Thanks for posting this, and glad you were able to solve it!

As you noted, those styles come from `components.css`. The one thing I wanted to flag is that the `style-editor.css` file is generated from all the same inputs as the main `style.css` file; the fundamental difference is that it disables Tailwind's Preflight—similar to a CSS reset—in the `tailwind.config.js` file. This helps prevent conflicts with block editor styles.

Hope that helps!
